### PR TITLE
Optimizing zh_CN translation

### PR DESCRIPTION
**Pull request by yuange1024**
Fixes: #

Explanation:  Optimizing zh_CN translation：
- Retain ”Gas“ instead of literal translation of ’天然气‘
- Fix Contract translation  to "合约"

### CODE REVIEW

**Comment by wanshan1024** [comment]
I have read the CLA Document and I hereby sign the CLA

File: app/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "message": "版本、技术支持中心和联系方式。"
   },
   "acceleratingATransaction": {
-    "message": "* 设定更高天燃气价格，加快交易完成进度，提高网络快速处理机率，但无法保证每次均能够实现提速。"
+    "message": "* 设定更高 Gas 价格，加快交易完成进度，提高网络快速处理机率，但无法保证每次均能够实现提速。"
```
Comment (DDDDDanica) [inline app/_locales/zh_CN/messages.json:9]: Suggestion: `"message": "* 设定更高的 Gas 价格，加快交易的完成进度，提高网络快速处理机率，但无法保证每次均能够实现提速。"`

File: app/_locales/zh_CN/messages.json [hunk 2]
```diff
@@ -150,7 +150,7 @@
     "message": "在CoinSwitch上购买"
   },
   "buyCoinSwitchExplainer": {
-    "message": "CoinSwitch是以最优惠的价格交换超过300种加密货币的一站式目的地。"
+    "message": "CoinSwitch是以最优惠的价格交换超过300种加密货币的一站式平台。"
```
Comment (DDDDDanica) [inline app/_locales/zh_CN/messages.json:153]: You need to add a space between english and chinese characters and numbers. 
```
"message": "CoinSwitch 是以最优惠的价格交换 300 多种加密货币的一站式平台。"
```

File: app/_locales/zh_CN/messages.json [hunk 3]
```diff
@@ -929,7 +929,7 @@
     "message": "助记词为12个单词"
   },
   "selectAHigherGasFee": {
-    "message": "请选择价格稍高的天然气费，加快交易处理速度。*"
+    "message": "请选择价格稍高的Gas，加快交易处理速度。*"
```
Comment (DDDDDanica) [inline app/_locales/zh_CN/messages.json]:
```
    "message": "请选择价格稍高的 Gas，来加快交易的处理速度。*"
```

File: app/_locales/zh_CN/messages.json [hunk 4]
```diff
@@ -980,10 +980,10 @@
     "message": "设置"
   },
   "showAdvancedGasInline": {
-    "message": "高级天然气控制"
+    "message": "高级Gas控制"
```
Comment (DDDDDanica) [inline app/_locales/zh_CN/messages.json]:
```
高级的 Gas 控制
```

File: app/_locales/zh_CN/messages.json [hunk 5]
```diff
@@ -980,10 +980,10 @@
...
   },
   "showAdvancedGasInlineDescription": {
-    "message": "请选择该选项，直接在发送和确认页面显示天然气价格和限制管理。"
+    "message": "请选择该选项，直接在发送和确认页面显示Gas价格和限制管理。"
```
Comment (DDDDDanica) [inline app/_locales/zh_CN/messages.json]:
```
直接在发送和确认页面显示 Gas 的价格和限制管理。
```

File: app/_locales/zh_CN/messages.json [hunk 6]
```diff
@@ -1058,7 +1058,7 @@
     "message": "3. 开始体验 dApps 和更多功能！"
   },
   "step3HardwareWalletMsg": {
-    "message": "使用您的硬件钱包，操作与以太坊账户制作相同。登录 dApps，发送 ETH ，购买和保存 ERC20 代币和诸如 CryptoKitties 等不可替代代币。"
+    "message": "使用您的硬件钱包，操作与以太坊账户制作相同。登录 dApps，发送 ETH ，购买和保存 ERC20 代币和非同质化代币如 CryptoKitties"
```
Comment (DDDDDanica) [inline app/_locales/zh_CN/messages.json:1061]: missing a period at the end of the sentence ~

**Comment by Gudahtt** [comment]
Hey @wanshan1024 , thanks for the contribution!

Do you have time to review each of these suggestions?

File: app/_locales/zh_CN/messages.json (hunk 1 above)
Comment (wanshan1024) [reply app/_locales/zh_CN/messages.json]: Thanks DDDDDanica.

File: app/_locales/zh_CN/messages.json [hunk 7]
```diff
@@ -413,19 +413,19 @@
     "message": "功能类型"
   },
   "gasLimitInfoTooltipContent": {
-    "message": "天然气限制即您个人希望购买的最大单位数量。"
+    "message": "gasLimit 即您个人希望付出的最大 Gas 数量。"
```
Comment (rekmarks) [inline app/_locales/zh_CN/messages.json:416]: There may be a more appropriate word than 限制, but we should definitely use the appropriate Mandarin for "limit" here:

```suggestion
Gas 的限制即您个人希望付出的最大 Gas 数量
```

In addition, we should add a message for `gasLimit`, which ought to be "Gas 的限制" or whatever is appropriate.

**Review by rekmarks — CHANGES_REQUESTED** [review]
One requested change, then this should be good to go!

File: app/_locales/zh_CN/messages.json (hunk 7 above)
Comment (astarinmymind) [reply app/_locales/zh_CN/messages.json:416]: I believe the following best captures the translation of: "Gas limit is the maximum amount of units of gas you are willing to spend."

```suggestion
    "message": "gasLimit 是您个人希望付出的最大 Gas 数量。"
```

Comment (rekmarks) [reply app/_locales/zh_CN/messages.json:416]: 越简单越好。

Comment (rekmarks) [reply app/_locales/zh_CN/messages.json:416]: Per [this website](https://cypherpunks-core.github.io/ethereumbook_zh/13.html), I believe we should make the following changes @wanshan1024:
```
  "gasLimit": {
    "message": "Gas 限制"
  },
  "gasLimitInfoTooltipContent": {
    "message": "Gas 限制是您个人希望付出的最大 Gas 数量。"
  },
  "gasLimitTooLow": {
    "message": "Gas 限制至少要 21000"
  },
```

Comment (rekmarks) [reply app/_locales/zh_CN/messages.json:416]: Please let me know if you disagree!

Comment (wanshan1024) [reply app/_locales/zh_CN/messages.json:416]: agree, thanks, I will change it later.

**Comment by wanshan1024** [comment]
translate 'Gas Limit' to 'Gas 上限'

File: app/_locales/zh_CN/messages.json [hunk 8]
```diff
@@ -412,20 +412,26 @@
...
   "gasLimitInfoTooltipContent": {
-    "message": "天然气限制即您个人希望购买的最大单位数量。"
+    "message": "Gas 上限是您个人希望付出的 Gas 数量上限。"
   },
   "gasLimitTooLow": {
-    "message": "Gas Limit 至少要 21000"
+    "message": "Gas 上限至少要 21000"
+  },
+  "gasPrice": {
+    "message": "Gas 价格 (GWEI)"
```
Comment (rekmarks) [inline app/_locales/zh_CN/messages.json:425]:
```suggestion
    "message": "Gas 价格 (gwei)"
```

**Review by rekmarks — APPROVED** [review]
真好，太谢谢你了！

**Pull request merged**